File: app/src/main/java/com/example/todonotesapp/DetailActivity.java
package com.example.todonotesapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class DetailActivity extends AppCompatActivity {

    TextView textViewtitle, textViewDescriptipn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        bindViews();
        setIntentData();
    }

    private void setIntentData() {
        Intent intent = getIntent();
        String title = intent.getStringExtra(AppConstant.TITLE);
        String desc = intent.getStringExtra(AppConstant.DESCRIPTION);

        textViewtitle.setText(title);
        textViewDescriptipn.setText(desc);
    }

    private void bindViews() {
        textViewtitle = findViewById(R.id.textViewTitle);
        textViewDescriptipn = findViewById(R.id.textViewDescription);
    }
}

File: app/src/main/java/com/example/todonotesapp/DetailActivity.kt
package com.example.todonotesapp

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class DetailActivity : AppCompatActivity() {
    val TAG = "DetailActivity"
    lateinit var textViewtitle : TextView
    lateinit var textViewDescriptipn : TextView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        bindViews()
        setIntentData()
    }

    private fun setIntentData() {
        val intent = intent //getIntent()
        val title = intent.getStringExtra(AppConstant.TITLE)
        val desc = intent.getStringExtra(AppConstant.DESCRIPTION)
        //setText()
        textViewtitle.text = title
        textViewDescriptipn.text = desc
    }

    private fun bindViews() {
        textViewtitle = findViewById(R.id.textViewTitle)
        textViewDescriptipn = findViewById(R.id.textViewDescription)

    }
}